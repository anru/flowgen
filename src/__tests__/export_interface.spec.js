import compiler from "../cli/compiler";
import beautify from "../cli/beautifier";

it("should handle export interface construction", () => {
  const ts = `
  export interface MyObj {
    state?: string
  }
`;

  const result = compiler.compileDefinitionString(ts);

  expect(beautify(result)).toMatchSnapshot();
});
