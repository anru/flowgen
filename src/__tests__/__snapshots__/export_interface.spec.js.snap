// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle export interface construction 1`] = `
"declare interface MyObj {
  state?: string;
}
"
`;
